G/Gnuplot/build_tarballs.jl

```diff
@@ -13,23 +12,27 @@ sources = [
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/gnuplot-*/
-
+cd $WORKSPACE/srcdir/gnuplot-*
 
 if [[ "${target}" == "${MACHTYPE}" ]]; then
     # Delete system libexpat to avoid confusion
     rm /usr/lib/libexpat.so*
 elif [[ "${target}" == *-mingw* ]]; then
     # This is needed because otherwise we get unusable binaries (error "The specified executable is not a valid application for this OS platform"). These come from CompilerSupportLibraries_jll:
```

Comment:
This comment doesn't make sense anymore without the code you deleted.

Reply:
Fixed.